tools/devops/device-tests/cambridge-ios-beta-device-tests.yml

```diff
@@ -0,0 +1,31 @@
+# Runs the device tests on iOS beta devices.
+parameters:
+- name: disableProviCleanup
+  type: boolean
+  default: false
+
+- name: clearProviCache
+  type: boolean
+  default: false
+
+variables:
+  - template: templates/common-variables.yml
+  - template: templates/cambridge-variables.yml
+
+resources:
+  repositories:
+  - repository: maccore
+    type: github
+    name: xamarin/maccore
+    ref: refs/heads/device-tests-yaml
+    endpoint: xamarin
```

Comment:
This part is also repeated in every yml script, can it be moved to the template?

Reply:
It cannot, resources cannot be moved to templates :/

Reply:
For reference the schema definition is
```yml
resources:
  pipelines: [ pipeline ]
  repositories: [ repository ]
  containers: [ container ]
```